Library/Homebrew/requirements/apr_requirement.rb

```diff
@@ -4,16 +4,22 @@ class AprRequirement < Requirement
   fatal true
   default_formula "apr-util"
 
-  # APR shipped in Tiger is too old, but Leopard+ is usable
-  satisfy(build_env: false) { MacOS.version > :leopard && MacOS::CLT.installed? }
+  # APR shipped in Tiger is too old, but Leopard+ is usable.
+  # The *-config scripts were removed in Sierra, which is widely breaking.
+  satisfy(build_env: false) do
+    next false if MacOS.version <= :leopard
```

Comment:
Leaves the current block and return the value `false`. `next` without `false` will return `nil` which is falsy but better to be more explicit.